packages/stylelint-config/README.md

```diff
@@ -19,27 +19,113 @@ npm install --save-dev @lmc-eu/stylelint-config stylelint-config-prettier
 > We assume you are using Prettier. That's why we also recommend adding
 > `stylelint-config-prettier` above.
 
-## Configurations
+New configuration extends community maintained config `stylelint-config-standard-scss`.
```

Comment:
Can you link those configs with their documentation sites or GitHub, please?

Reply:
Links added.